pkg/pillar/cmd/zedagent/handlemetrics.go

```diff
@@ -1018,7 +1018,7 @@ func hardwareHealthTimerTask(ctx *zedagentContext, handleChannel chan interface{
 	interval := time.Duration(hardwareHealthInterval) * time.Second
 	if retry {
 		log.Noticef("Initial publishHardwareHealth failed; switching to short timer")
-		interval = shortTime
+		interval = shortTime * time.Second
```

Comment:
I would suggest to make this part of the `shortTime` const definition to avoid repeating this mistake.

Reply:
But `shortTime` is also used with `updateTaskTimer()`, which already expects the value in seconds, so the `* time.Second` is only required in this line....

Reply:
How about introducing another variable to make this more clear? The place it was copied from has
	const shortTime = 120 // Two minutes
	certInterval := ctx.globalConfig.GlobalValueInt(types.CertInterval)
	if retry {
		log.Noticef("Initial getCertsFromController failed; switching to short timer")
		certInterval = shortTime
	}
	interval := time.Duration(certInterval) * time.Second

The replace the certInterval name with something like hhInterval for this function.

Reply:
I liked @eriknordmark's suggestion, the PR is updated, please folks, take a look now....

Reply:
ok, in order to make it more clear and avoid confusions, I've changed the name of variable to explicitly say it's in seconds, as suggested by @OhmSpectator .....